### PR TITLE
MGMT-853: Use PVs for mariadb and scality storage

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: tools/create_default_s3_bucket.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import os
 import utils
 import boto3
+import sys
```
Comment (ronniel1) [inline tools/create_default_s3_bucket.py:4]: Where is this used?

Comment (avishayt) [reply tools/create_default_s3_bucket.py:4]: Not used by this commit - it was missing previously

**Comment by avishayt** [comment]
> Will this also work in minikube?

Yes, that's where I tested it.  Basically it creates /mnt/data/mariadb and /mnt/data/scality, and maps the directories to where the applications in the containers put the data.

**Comment by filanov** [comment]
@avishayt probably need to add the cleanup to `clear-deployment`

**Comment by avishayt** [comment]
> @avishayt probably need to add the cleanup to `clear-deployment`

Good call - added

**Pull request merged**